core/src/main/java/org/fao/geonet/kernel/Thesaurus.java

```diff
@@ -738,10 +738,6 @@ private void retrieveThesaurusTitle(Path thesaurusFile, String defaultTitle, boo
                 this.defaultNamespace = DEFAULT_THESAURUS_NAMESPACE;
             }
 
-            if (!this.defaultNamespace.endsWith("#")) {
```

Comment:
it looks like there is some usages https://github.com/geonetwork/core-geonetwork/blob/master/core/src/main/java/org/fao/geonet/kernel/KeywordBean.java#L408-L441 but not sure if needed or not.

Reply:
The management in KeywordBean is indeed necessary to manage when the code includes also the namespace. My change should be safe and impact only the namespace shown without code. On your suggestion, I've added an extra check on L363.  Thanks for the feedback